Gems/Camera/Code/Source/CameraComponentController.cpp

```diff
@@ -154,6 +154,11 @@ namespace Camera
         m_shouldActivateFn = shouldActivateFunction;
     }
 
+    void CameraComponentController::SetIsLockedFunction(AZStd::function<bool()> isLockedFunction)
+    {
+        m_isLockedFn = isLockedFunction;
```

Comment:
nit: As we're passing the std::function by value and storing it we can add `AZStd::move` here to prevent an additional copy.